app/controllers/taggings_controller.rb

```diff
@@ -55,12 +58,17 @@ def update
       render :show, locals: { tagging_update: tagging_update }
     end
   rescue GdsApi::HTTPConflict
-    redirect_to :back, danger: "Somebody changed the tags before you could. Your changes have not been saved."
+    redirect_back(
+      fallback_location: tagging_path(content_item.content_id),
+      danger: "Somebody changed the tags before you could. Your changes have not been saved."
+    )
   end
 
 private
 
   def lookup_params
-    params[:content_lookup_form] || { base_path: "/#{params[:slug]}" }
+    params.require(:content_lookup_form).permit(:base_path)
+  rescue ActionController::ParameterMissing
+    { base_path: "/#{params[:slug]}" }
```

Comment:
We don't really require the `content_lookup_form` param so maybe we could do this?

```rb
params
  .fetch(:content_lookup_form, { base_path: "/#{params[:slug]}" })
  .permit(:base_path)
```